fdbcli/TenantCommands.actor.cpp

```diff
@@ -942,9 +1019,13 @@ std::vector<const char*> tenantHintGenerator(std::vector<StringRef> const& token
 		static std::vector<const char*> opts = { "<ID>" };
 		return std::vector<const char*>(opts.begin() + tokens.size() - 2, opts.end());
 	} else if (tokencmp(tokens[1], "list") && tokens.size() < 7) {
-		static std::vector<const char*> opts = {
-			"[BEGIN]", "[END]", "[limit=LIMIT]", "[offset=OFFSET]", "[state=<STATE1>,<STATE2>,...]"
-		};
+		static std::vector<const char*> opts = { "[BEGIN]",
+			                                     "[END]",
+			                                     "[limit=LIMIT]",
+			                                     "[offset=OFFSET]",
+			                                     "[state=<STATE1>,<STATE2>,...]",
+			                                     "[tenant_group=TENANT_GROUP]",
+			                                     "JSON" };
```

Comment:
```suggestion
			                                     "[JSON]" };
```

Reply:
We should also update the `tenantGenerator` function to generate the possible tokens here: `limit=`, `offset=`, `state=`, `tenant_group=`, and `JSON`.